services-camel-app/src/main/java/edu/unc/lib/boxc/services/camel/images/ImageDerivativeProcessor.java

```diff
@@ -47,6 +48,21 @@ public static boolean allowedImageType(Exchange exchange) {
         return true;
     }
 
+    /**
+     * Run jp24u kdu_compress command and generate image derivatives
+     * @param exchange
+     */
+    public void runJp24u(Exchange exchange) {
```

Comment:
I know this is fairly short, but my suggestion would be to make a new Jp2Processor for this. That would keep the scope of ImageDerivativeProcessor narrower, and would mean we could call `.bean(jp2Processor)` in the camel route, since this method would become "process"

We should also have a test for this method/processor. I'm not sure it'd be practical to actually test that jp24u works in boxc since it requires so many dependencies including one that requires a license, but we can at least mock `CLIMain.main` (search the code base for `mockStatic` for examples)